ext/calendar/easter.c

```diff
@@ -48,11 +53,16 @@ static void _cal_easter(INTERNAL_FUNCTION_PARAMETERS, bool gm)
 		}
 	}
 
-	if (gm && (year<1970 || year>2037)) {				/* out of range for timestamps */
+	if (gm && (year<1970 || year>2037) && ZEND_LONG_MAX == INT32_MAX) { /* out of range for timestamps on 32bit systems */
 		zend_argument_value_error(1, "must be between 1970 and 2037 (inclusive)");
 		RETURN_THROWS();
 	}
 
+	if (gm && (year<1970 || year>2000000000)) {             /* timestamp range is between 1970 and the year 2.000.000.000 */
```

Comment:
@javiereguiluz

I fixed the spelling and also added a check here to make sure that this won't lead to an overflow. I think this would be ready for another review!